app/models/assets/import.rb

```diff
@@ -85,11 +85,11 @@ def refresh(fact_changes = nil)
       self
     end
 
-    def refresh_from_remote(remote_asset:, fact_changes: nil)
```

Comment:
The import action resulted in two steps, import and refresh. The facts associated with the former got nuked by the refresh, and the step never got resolved.

Here we pass in the import step such that we can use it exclusively.